bin/test-code-style

```diff
@@ -2,14 +2,7 @@
 
 . "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/scriptLib"
 
-runSbt  scalariformFormat \
-        test:scalariformFormat \
-        multi-jvm:scalariformFormat \
-        checkHeaders \
-        test:checkHeaders \
-        multi-jvm:checkHeaders \
-        validateDependencies \
-        mimaCheckOneAtATime
+runSbt scalariformFormat test:scalariformFormat multi-jvm:scalariformFormat validateDependencies mimaCheckOneAtATime
```

Comment:
Did you mean to make this change?

Reply:
no, I didn't. Hold one with everything here because I'm afraid that the merge I did to resolve conflicts went totally wrong.

The reason why the `internal.*` stopped working is that the mima plugin version was reverted somehow. I need to review everything now.